doc/rst/07_api_reference.rst

```diff
@@ -0,0 +1,60 @@
+|newpage|
+
+.. _lib_camera_api_reference:
+
+API Reference
+=============
+
+.. contents::
+   :local:
+   :class: this-will-duplicate-information-and-it-is-still-useful-here
+
+Camera
+------
+
+.. doxygengroup:: camera_main
+   :project: lib_camera
+   :members:
+
+Sensors
+-------
+
+.. doxygengroup:: camera_sensors
+   :project: lib_camera
+   :members:
+
+MIPI
+----
+
+.. doxygengroup:: camera_mipi
+   :project: lib_camera
+   :members:
```

Comment:
Do we really need to expose this here? it's a mipi conf struct and an init api, both used internally in camera_main